game/game.c

```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+enum figure_num_t {
+        PAPER = 0,
+        ROCK,
+        SCISSORS,
+        MAX_FIGURE_NUM
+};
+
+static const char FIGURE_SYM[3] = { 'p', 'r', 's' };
+
+static const char *FIGURE_FULL_NAME[] = { "paper", "rock", "scissors" };
+
+int main(void)
+{
+        enum figure_num_t player_num;
+        enum figure_num_t bot_num;
+        char bot_fig;
+        char player_fig;
+        bool player_step_flag;
+
+        printf("Press \"Ctrl + C\" for exit\n");
+
+        while(1) {
+
+                bot_num = rand()%3;
+
+                switch (bot_num) {
+                        case PAPER:     bot_fig = FIGURE_SYM[PAPER];    break;
+                        case ROCK:      bot_fig = FIGURE_SYM[ROCK];     break;
+                        case SCISSORS:  bot_fig = FIGURE_SYM[SCISSORS]; break;
+                }
```

Comment:
In my opinion it can be transformed to oneliner:
bot_fig = FIGURE_SYM[bot_num];